.ci/generate.sh

```diff
@@ -35,6 +35,15 @@ fi
 # move the generated CRD to the same location the operator-sdk places
 mv deploy/crds/jaegertracing.io_jaegers.yaml deploy/crds/jaegertracing.io_jaegers_crd.yaml
 
+# the controller-gen will generate a list of CRDs, but the operator-sdk tooling expects
+# a single item
+# the proper solutions are, in order:
+# 1) find a controller-gen switch that makes it write only one CRD. Such a switch doesn't exist yet: https://git.io/JvX5D
+# 2) use a YAML command line tool to get the first item from the file
+# 3) chop off the first two lines of the file
+# the last option is the easiest to implement for now, also because `tail` is found everywhere
+echo "$(tail -n +3 deploy/crds/jaegertracing.io_jaegers_crd.yaml)" > deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
Actually maybe more robust would be to remove lines with `---`. Reason being they can remove empty line and this will chop off the first lines of the CRD

Reply:
Your suggestion gave me another idea: how about I positively check that the first line matches with what we expect it to be? This way, changes to the controller-gen will be caught right away.

Reply:
+1

Reply:
PR updated with this suggestion.